docs/source/releases/HEADER

```diff
@@ -0,0 +1,4 @@
+.. dropdown:: Distribution Statement
+
+ | # # # This source code is protected under the license referenced at
+ | # # # https://github.com/NRLMMD-GEOIPS.
```

Comment:
Is this still required?  Is this part of brassy?

Reply:
Does it do anything?

Reply:
I think we can get rid of that. I added that when initially testing brassy as I thought we needed it but after talking to Gwyn it sounds like we don't.